services/app-web/src/constants/statutoryRegulatoryAttestation.ts

```diff
@@ -0,0 +1,12 @@
+const StatutoryRegulatoryAttestation = {
+    YES: 'Yes the contract fully complies with all applicable requirements.',
+    NO: 'No the contract does not fully comply with all applicable requirements.',
```

Comment:
```suggestion
    YES: 'Yes, the contract fully complies with all applicable requirements',
    NO: 'No, the contract does not fully comply with all applicable requirements',
```

matching up the text with designs 💅